app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -98,6 +129,14 @@ def forward_to_mas?
       Flipper.enabled?(:rrd_mas_disability_tracking)
   end
 
+  def new_pact_related_disability?
+    disabilities.any? do |disability|
+      (RapidReadyForDecision::Constants::PACT_CLASSIFICATION_CODES.include? disability['classificationCode']) &&
+        disability['disabilityActionType'] == 'NEW'
```

Comment:
Opinion: I would reverse this because checking for `NEW` is faster than checking for set inclusion:
```suggestion
      disability['disabilityActionType'] == 'NEW' && (RapidReadyForDecision::Constants::PACT_CLASSIFICATION_CODES.include? disability['classificationCode'])        
```